### PR TITLE
lab3 update

### PR DESCRIPTION
**Pull request by vidartf**
Switches to use lab v3.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        python-version: '3.7'
+        architecture: 'x64'
+    - name: Install dependencies
+      run: python -m pip install jupyterlab
+    - name: Build and install the extension
+      run: |
+        pip install .
+
+        # This doesn't work???
+        # jupyter labextension list | grep -ie "jupyterlab-kernelspy.*enabled.*ok" -
```
Comment (jtpio) [inline .github/workflows/build.yml:31]: Maybe something like in the cookiecutter would work? (or a similar trick)

https://github.com/jupyterlab/extension-cookiecutter-ts/blob/cb9ca8c37bf1c5c28be61b31e6fcd0cf60487a5f/.github/workflows/main.yml#L52-L53

Comment (vidartf) [reply .github/workflows/build.yml:31]: Note that this is commented out because I couldn't get grep to work correctly 😢

Comment (vidartf) [reply .github/workflows/build.yml:31]: ah, I misread what you wrote, I'll give that one a try.

Comment (vidartf) [reply .github/workflows/build.yml:31]: But yeah, I did actually find the problem on Friday but forgot about it: grep should be running against stderr in this case (where log entries go), as that code fixes 👍

**Pull request merged**